flowtype.js

```diff
@@ -1 +1 @@
-module.exports = require('./packages/flowtype.json');
\ No newline at end of file
+module.exports = require('./packages/flowtype.json');
```

Comment:
mag die file niet weg dan ?

Reply:
I prefer not to remove this one, so users can still choose flowtype with a more custom approach.